code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,234 @@
 			new /obj/item/clothing/mask/luchador(src)
 		if(100)
 			new /obj/item/clothing/head/bearpelt(src)
+		if(101 to 105)
+			var/ctype = pick(subtypesof(/datum/supply_pack))
+			var/datum/supply_pack/pack = new ctype()
+			for(var/item in pack.contains)
+				new item(src)
+			qdel(pack)
+		if(106)
+			new /obj/item/reagent_containers/glass/bottle/mannitol(src)
+			new /obj/item/storage/pill_bottle/neurine(src)
+			new /obj/item/organ/brain(src)
+		if(107)
+			new /obj/item/reagent_containers/glass/bottle/brainrot(src)
+			new /obj/item/reagent_containers/glass/bottle/gbs(src)
+			new /obj/item/reagent_containers/glass/bottle/tuberculosis(src)
+			new /obj/item/reagent_containers/glass/bottle/tuberculosiscure(src)
+			new /obj/item/reagent_containers/glass/bottle/wizarditis(src)
```

Comment:
All of these viruses will result in large amounts of the crew dying guaranteeing the round to be ended, GBS and wizards are also nuke ops and wizard items respectively.